pages/services/beta-storage/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: layout.pug
+navigationTitle:  Beta Storage
+title: Beta Storage
+menuWeight: 210
+enterprise: true
+excerpt:
+---
+
+
+DC/OS Beta Storage Service (DSS) is a service that manages storage in DC/OS.
```

Comment:
I'd void having Beta in the content so that we don't need to go back and change the content when GA. It's fine to have Beta in the title.

This is also inline with other service is doing (e.g., kubernetes)